background-process.js

```diff
@@ -1,34 +1,28 @@
-var fs = require('fs')
-var Path = require('path')
-var electron = require('electron')
+const electron = require('electron')
+const Client = require('ssb-client')
+const scuttleshell = require('scuttle-shell')
 
-console.log('STARTING SBOT')
+// Get config options from depject
+const config = require('./config').create().config.sync.load()
 
-var createSbot = require('scuttlebot')
-  .use(require('scuttlebot/plugins/master'))
-  .use(require('scuttlebot/plugins/gossip'))
-  .use(require('scuttlebot/plugins/replicate'))
-  .use(require('scuttlebot/plugins/invite'))
-  .use(require('scuttlebot/plugins/local'))
-  .use(require('scuttlebot/plugins/logging'))
-  .use(require('ssb-about'))
-  .use(require('ssb-backlinks'))
-  .use(require('ssb-blobs'))
-  .use(require('ssb-chess-db'))
-  .use(require('ssb-ebt'))
-  .use(require('ssb-friends'))
-  .use(require('ssb-meme'))
-  .use(require('ssb-private'))
-  .use(require('ssb-query'))
-  .use(require('ssb-search'))
-  .use(require('ssb-unread'))
-  .use(require('ssb-ws'))
-  // .use(require('ssb-mutual')) // this is has recursion problems atm
+// Check if scuttle-shell is already running
+// TODO - make this check for scuttle-shell specifically (and not just an sbot)
```

Comment:
@mixmix I'm also not sure about this comment.

How would the shell be different from another sbot with the same plugins?